src/memory_model.ts

```diff
@@ -73,7 +74,8 @@ export class MemoryModel {
 
         this.svg.setAttribute("width", options.width || 800);
         this.svg.setAttribute("height", options.height || 800);
-        this.rough_svg = rough.svg(this.svg);
+        this.seed = options.seed || 0;
```

Comment:
As of right now, the `|| 0` is unnecessary on constructor level, because all calls to the constructor already specifies a numeric seed. However, I'm still inclined to include it here just in case future calls don't handle it.

Reply:
I'm fine with the `|| 0` here to keep the type `number`. However, I don't think you need to include `|| 0` in all of the other places. If `options.seed` is `undefined` we can still set `this.seed` to `0`.